lib/pbench/server/api/resources/upload_api.py

```diff
@@ -79,26 +79,26 @@ def put(self, controller):
         md5sum = request.headers.get("Content-MD5")
 
         if not self.allowed_file(filename):
-            self.logger.debug(
+            self.logger.warning(
                 f"Tarfile upload: Bad file extension received for file {filename}"
             )
             abort(400, message="File extension not supported. Only .xz")
 
         try:
             content_length = int(request.headers.get("Content-Length"))
         except ValueError:
-            self.logger.debug(
+            self.logger.warning(
                 f"Tarfile upload: Invalid content-length header, not an integer for file {filename}"
             )
             abort(400, message="Invalid content-length header, not an integer")
         except Exception:
-            self.logger.debug(
+            self.logger.exception(
```

Comment:
I tend to agree with an `exception` level for a miscellaneous `Exception` ... however looking at this it seems clear that the message here is assuming that it was a `KeyError` for a missing header, and if it's really something else the messaging will be misleading. I think the code here should be an `except KeyError as e` and report the `str(e)` (probably `"Content-Length"`), with a `warning` level.

If we really think there's a reason to anticipate some other generic `Exception` here, we should add a separate `except Exception` with a 500 abort and an `exception` level log with a more generic description. But I don't think that's very likely...